server_src/lib/serverCommands.js

```diff
@@ -4,8 +4,8 @@ const { isNodemon } = require("../utils/env.utils.js");
 const { lookpath } = require("lookpath");
 
 const {
-  doWeHaveMissingPackages,
-  installMissingNpmDependencies,
+  returnListOfMissingPackages,
+  installNpmDependencies,
```

Comment:
These names are more understandable no? xD

Best be... *shakes fist*

Reply:
I can save this line of code and comment and show it to you half a year later? 🦆

Reply:
!RemindMe 1 year